client/src/Components/Filters/FilterFunctions.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { Row } from "../../../public/stylesheets/styles.js";
+import styled from "styled-components";
+import Button from "@mui/material/Button";
+import Select from "@mui/material/Select";
+import MenuItem from "@mui/material/MenuItem";
+import InputLabel from "@mui/material/InputLabel";
+
+class FilterFunctions extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      industry: "",
+
+      isRemote: 2,
+
+      employmentType: "",
+
+      minDistance: 5,
+
+      maxDistance: "",
+    };
+
+    this.setIndustry = this.setIndustry.bind(this);
+    this.setSite = this.setSite.bind(this);
+    this.setEmployment = this.setEmployment.bind(this);
+    this.setMaxDist = this.setMaxDist.bind(this);
+  }
+
+  setIndustry(event) {
```

Comment:
All of these `set<insert>` functions can potentially be condensed to a single `setValue()` function that destructures the event object to pull the target information. However, you will need to add a `name` property to each of the elements that it will be used on

Example:
`
setValue({ target: { name, value } }) {
this.setState({ [name]: value})
}
`